### PR TITLE
Update types of standards

### PR DESCRIPTION
**Pull request by rogueassasin1729**
Updated the Types of Standard tab to elucidate the different types of EIPs(Standard, Meta & Informational) , and list down the sub-categories of Standard Track EIP.

### CODE REVIEW

**Review by minimalsm — APPROVED** [review]
LGTM :-) Thanks for the PR @rogueassasin1729 

I've asked for a double-check of this in the cat herders Discord but we'll get it merged in soon.

File: src/content/developers/docs/standards/index.md [hunk 1]
```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
 ## Types of standards {#types-of-standards}
 
-Certain EIPs relate to application-level standards (e.g. a standard smart-contract format), which are introduced as [Ethereum Requests for Comment (ERC)](https://eips.ethereum.org/erc). Many ERCs are critical standards used widely across the Ethereum ecosystem.
+There are 3 types of EIPs:
+- [Standard Track](https://eips.ethereum.org/)
+- [Meta Track](https://eips.ethereum.org/meta)
+- [Informational Track](https://eips.ethereum.org/informational)
 
-- [List of ERCs](https://eips.ethereum.org/erc)
+Further, the Standard Track is sub-divided into 4 categories:
```
Comment (Pandapip1) [inline src/content/developers/docs/standards/index.md:31]:
```suggestion
Furthermore, the Standard Track is subdivided into 4 categories:
```
Nit.

File: src/content/developers/docs/standards/index.md [hunk 2]
```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
 
 ## Types of standards {#types-of-standards}
 
-Certain EIPs relate to application-level standards (e.g. a standard smart-contract format), which are introduced as [Ethereum Requests for Comment (ERC)](https://eips.ethereum.org/erc). Many ERCs are critical standards used widely across the Ethereum ecosystem.
+There are 3 types of EIPs:
+- [Standard Track](https://eips.ethereum.org/)
```
Comment (Pandapip1) [inline src/content/developers/docs/standards/index.md:27]:
```suggestion
- [Standards Track](https://eips.ethereum.org/): describes any change that affects most or all Ethereum implementations
```

File: src/content/developers/docs/standards/index.md [hunk 3]
```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
+- [Meta Track](https://eips.ethereum.org/meta)
```
Comment (Pandapip1) [inline src/content/developers/docs/standards/index.md:28]:
```suggestion
- [Meta Track](https://eips.ethereum.org/meta): describes a process surrounding Ethereum or proposes a change to a process
```

File: src/content/developers/docs/standards/index.md [hunk 4]
```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
+- [Informational Track](https://eips.ethereum.org/informational)
```
Comment (Pandapip1) [inline src/content/developers/docs/standards/index.md:29]:
```suggestion
- [Informational Track](https://eips.ethereum.org/informational): describes an Ethereum design issue or provides general guidelines or information to the Ethereum community
```

File: src/content/developers/docs/standards/index.md [hunk 5]
```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
 
-- [List of ERCs](https://eips.ethereum.org/erc)
+Further, the Standard Track is sub-divided into 4 categories:
+
+- [Core](https://eips.ethereum.org/core)
```
Comment (Pandapip1) [inline src/content/developers/docs/standards/index.md:33]:
```suggestion
- [Core](https://eips.ethereum.org/core): improvements requiring a consensus fork
```

File: src/content/developers/docs/standards/index.md [hunk 6]
```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
+- [Networking](https://eips.ethereum.org/networking)
```
Comment (Pandapip1) [inline src/content/developers/docs/standards/index.md:34]:
```suggestion
- [Networking](https://eips.ethereum.org/networking): improvements around devp2p and Light Ethereum Subprotocol, as well as proposed improvements to network protocol specifications of whisper and swarm.
```

File: src/content/developers/docs/standards/index.md [hunk 7]
```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
+- [Interface](https://eips.ethereum.org/interface)
```
Comment (Pandapip1) [inline src/content/developers/docs/standards/index.md:35]:
```suggestion
- [Interface](https://eips.ethereum.org/interface): improvements around client API/RPC specifications and standards, and certain language-level standards like method names and contract ABIs.
```

File: src/content/developers/docs/standards/index.md [hunk 8]
```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
+- [ERC](https://eips.ethereum.org/erc)
```
Comment (Pandapip1) [inline src/content/developers/docs/standards/index.md:36]:
```suggestion
- [ERC](https://eips.ethereum.org/erc): application-level standards and conventions
```

File: src/content/developers/docs/standards/index.md [hunk 9]
```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
+
+The main difference in the above categories of Standards Track EIP is where it's deployed. For instance, ERCs are deployed at the application level and therefore don't need to be adopted by all participants. However, Core Track EIP is deployed at the protocol level and requires a broader consensus in the community as all core EIPs must be widely adopted (i.e. all the nodes must upgrade to remain part of the network)
```
Comment (Pandapip1) [inline src/content/developers/docs/standards/index.md:38]:
```suggestion
More detailed information on these different types and categories can be found in [EIP-1](https://eips.ethereum.org/EIPS/eip-1#eip-types)
```

File: src/content/developers/docs/standards/index.md [hunk 10]
```diff
@@ -23,9 +23,19 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
 
 ## Types of standards {#types-of-standards}
 
-Certain EIPs relate to application-level standards (e.g. a standard smart-contract format), which are introduced as [Ethereum Requests for Comment (ERC)](https://eips.ethereum.org/erc). Many ERCs are critical standards used widely across the Ethereum ecosystem.
+There are 3 types of EIPs:
```
Comment (Pandapip1) [inline src/content/developers/docs/standards/index.md:26]:
```suggestion
There are 3 types of EIPs:

```
Markdown sometimes doesn't like a list right after text, so adding a new line is helpful here.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

**Comment by minimalsm** [comment]
Thanks @rogueassasin1729 and @Pandapip1 <3

Both please remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @rogueassasin1729 for docs

**Comment by minimalsm** [comment]
@all-contributors please add @Pandapip1 for docs

File: src/content/developers/docs/standards/index.md [hunk 11]
```diff
@@ -23,9 +23,20 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
 
-- [List of ERCs](https://eips.ethereum.org/erc)
+- [Standards Track](https://eips.ethereum.org/): describes any change that affects most or all Ethereum implementations
```
Comment (EtherWorldCo) [inline src/content/developers/docs/standards/index.md:28]: I will rather call it "EIPs website".
IMHO, Pointing "Standard Track" to  https://eips.ethereum.org/ may not be appropriate.

File: src/content/developers/docs/standards/index.md [hunk 12]
```diff
@@ -23,9 +23,20 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
+- [Meta Track](https://eips.ethereum.org/meta): describes a process surrounding Ethereum or proposes a change to a process
```
Comment (EtherWorldCo) [inline src/content/developers/docs/standards/index.md:29]: "Meta Track" isn't commonly used. Probably "Meta EIPs" will be a better fit.

File: src/content/developers/docs/standards/index.md [hunk 13]
```diff
@@ -23,9 +23,20 @@ Typically standards are introduced as [Ethereum Improvement Proposals](/eips/) (
...
+- [Informational Track](https://eips.ethereum.org/informational): describes an Ethereum design issue or provides general guidelines or information to the Ethereum community
```
Comment (EtherWorldCo) [inline src/content/developers/docs/standards/index.md:30]: Same as comment above. IMO, "Informational EIPs" will be better.

File: src/content/developers/docs/standards/index.md (hunk 11 above)
Comment (Pandapip1) [reply src/content/developers/docs/standards/index.md:28]: Yea, that's definitely a typo. Fixing

File: src/content/developers/docs/standards/index.md (hunk 12 above)
Comment (Pandapip1) [reply src/content/developers/docs/standards/index.md:29]: This is the technical term and is the correct term to use here.

File: src/content/developers/docs/standards/index.md (hunk 13 above)
Comment (Pandapip1) [reply src/content/developers/docs/standards/index.md:30]: Likewise, this is the correct term to use here.

**Pull request merged**